SimCalorimetry/HGCalSimProducers/src/HGCDigitizer.cc

```diff
@@ -582,6 +602,9 @@ uint32_t HGCDigitizer::getType() const {
     case DetId::HGCalHSc:
       idx = 2;
       break;
+    case DetId::Forward:
+      idx = 0;
```

Comment:
does it make sense to assign index 0 to both EE and HFNose?

Reply:
Changed it to 4